tests/system/feature_store/test_spark_engine.py

```diff
@@ -58,6 +60,7 @@ class TestFeatureStoreSparkEngine(TestMLRunSystem):
     pq_source = "testdata.parquet"
     pq_target = "testdata_target"
     csv_source = "testdata.csv"
+    run_local = False
```

Comment:
seems like the periodic system tests will still run remotely, I guess this is mainly for development test purposes.
I suggest adding a docstring to the class with guideline on how the developer should test its tests when developing and the possible features it can use

Reply:
That's correct. Done.